commit 09971062c6c83e66c23d41c86321bbd6353725dc
Author: Bernhard Rosenkränzer <bero@lindev.ch>
Date:   Sat Aug 15 04:48:06 2020 +0200

    Don't whine about missing iwlwifi debugging firmware
    
    iwl-debug-yoyo.bin doesn't seem to be publicly available, and it's
    not required for the driver to work -- so shut up about it.
    
    Signed-off-by: Bernhard Rosenkränzer <bero@lindev.ch>

diff --git a/drivers/net/wireless/intel/iwlwifi/iwl-dbg-tlv.c b/drivers/net/wireless/intel/iwlwifi/iwl-dbg-tlv.c
index 27116c7d3f4f8..9ce7207d9ec5b 100644
--- a/drivers/net/wireless/intel/iwlwifi/iwl-dbg-tlv.c
+++ b/drivers/net/wireless/intel/iwlwifi/iwl-dbg-tlv.c
@@ -480,7 +480,7 @@ void iwl_dbg_tlv_load_bin(struct device *dev, struct iwl_trans *trans)
 	if (!iwlwifi_mod_params.enable_ini)
 		return;
 
-	res = request_firmware(&fw, "iwl-debug-yoyo.bin", dev);
+	res = firmware_request_nowarn(&fw, "iwl-debug-yoyo.bin", dev);
 	if (res)
 		return;
 
